complaints/models.py

```diff
@@ -116,25 +118,30 @@ def validate_timings(self, avail_start_time, avail_end_time):
         if self.bhavan in self.GIRLS_BHAVANS:
             for start_time, end_time in self.GIRLS_TIME_LIMITS:
                 if avail_start_time >= start_time and avail_end_time <= end_time:
-                    break
+                    return
             else:
                 raise ValidationError(
                     "Girls' Hostel is only open for limited time slots. Your given time does not fit in them"
                 )
+        for start_time, end_time in self.TECHNICIAN_WORK_HOURS:
+            if avail_start_time >= start_time and avail_end_time <= end_time:
+                break
```

Comment:
Don't be too strict with the timings.Even if the availibility timings are from 9 am to 2 pm they should be accepted.But if the user enters the timings which completely lie in the technicians lunch hour ,then only an error message should b sent.

Reply:
I don't know fuzzy logic dude. Give me hard values

Reply:
This isn't any fuzzy logic dude!!! My comment is quite easy to understand.Understand it to contemplate the hard rules yourself!

Reply:
I don't get it. I need some help senpai